sample-cap/package.json

```diff
@@ -6,16 +6,34 @@
   "repository": "https://github.com/sap/ai-sdk-js",
   "private": true,
   "dependencies": {
-    "@sap-ai-sdk/ai-api": "workspace:^",
-    "@sap-ai-sdk/foundation-models": "workspace:^",
-    "@sap-ai-sdk/orchestration": "workspace:^",
-    "@sap/cds": "^8",
-    "express": "^4"
+    "@sap-ai-sdk/ai-api": "^1",
+    "@sap-ai-sdk/foundation-models": "^1",
+    "@sap-ai-sdk/orchestration": "^1",
+    "express": "^4",
+    "@sap/xssec": "^4",
+    "@sap/cds": "^8.4.1"
+  },
+  "devDependencies": {
+    "@cap-js/cds-types": "^0.7.0"
   },
   "scripts": {
+    "prebuild": "rm -rf dist",
+    "build": "pnpm cds-build && pnpm compile && pnpm cleanup",
+    "cds-build": "cds build --production",
+    "compile": "pnpm rebuild && tsc",
```

Comment:
Fyi, since sometimes types cannot be resolved correctly from `@sap/cds` or `@cap-js/cds-types` due to missing link, running `pnpm rebuild` can resolve the issue. See https://github.com/cap-js/cds-types/issues/294